README.md

```diff
@@ -1,7 +1,56 @@
-# Scoop [![Build status](https://ci.appveyor.com/api/projects/status/05foxatmrqo0l788?svg=true)](https://ci.appveyor.com/project/lukesampson/scoop) [![Gitter chat](https://badges.gitter.im/lukesampson/scoop.png)](https://gitter.im/lukesampson/scoop)
+<p align="center">
+<img src="scoop.png" alt="Long live Scoop!"/>
+</p>
+<p align="center">
+<b><a href="https://github.com/lukesampson/scoop#what-does-scoop-do">Features</a></b>
+|
+<b><a href="https://github.com/lukesampson/scoop#installation">Installation</a></b>
+|
+<b><a href="https://github.com/lukesampson/scoop/wiki">Documentation</a></b>
+</p>
+
+- - -
+<p align="center" >
+    <a href="https://github.com/lukesampson/scoop">
+        <img src="https://img.shields.io/github/languages/code-size/lukesampson/scoop.svg" alt="Code Size" />
+    </a>
+    <a href="https://github.com/lukesampson/scoop">
+        <img src="https://img.shields.io/github/repo-size/lukesampson/scoop.svg" alt="Repository size" />
+    </a>
+    <a href="https://ci.appveyor.com/project/lukesampson/scoop">
+        <img src="https://ci.appveyor.com/api/projects/status/05foxatmrqo0l788?svg=true" alt="Build Status" />
```

Comment:
Is 05foxatmrqo0l788 correct?

Reply:
Well, that's whats in the current README

Reply:
Yep still working. It's also in the current README.